apps/condo/domains/notification/utils/serverSchema/index.js

```diff
@@ -151,13 +153,44 @@ async function checkMessageTypeInBlackList (context, message) {
         if (isMessageTypeInUserBlackList) return { error: MESSAGE_TYPE_IN_USER_BLACK_LIST }
     }
 
+    //TODO(Kekmus) think more about it maybe in some cases it doesnt work
+    if (message.meta.data && message.meta.data.B2CAppId) {
```

Comment:
```suggestion
    if (get(message['meta', 'data', 'B2CAppId'])) {
```
The `get` function checks for a full path to the target field.